### DISCUSSION TITLE
Most upto-date recipe for Streaming ASR

### DISCUSSION THREAD

**Discussion by sangeet2020** [espnet/espnet#5820] [Q&A]
Hi

I'm exploring ESPnet for training streaming ASR models, and I am particularly interested in using recipes that support real-time ASR with punctuated text. While I have come across various options like RNN-T, Transformer, and Conformer models, I'm uncertain which recipes are the most current and can directly handle punctuated text in the training phase. 

Could anyone guide me on which recipes are best suited for this, or share any recent updates on supporting punctuated text in streaming ASR training?

Thank You

**Comment by sw005320** [comment]
- for the punctuated text, please check the OWSM recipe (https://github.com/espnet/espnet/tree/master/egs2/owsm_v3.1/s2t1)
- for streaming ASR, `egs2/aishell/asr1/run_streaming.sh`, `egs2/laborotv/asr1/run_streaming.sh`, and `egs2/tedlium2/asr1/run_streaming.sh` are still good examples.

However, we don't prepare a combination of both.

**Comment by sangeet2020** [comment]
Thank You @sw005320 for answering.
